td/tracking/forms.py

```diff
@@ -370,6 +368,15 @@ def check_text_input(form, field_name):
         return escape(text)
 
 
+def determine_widget(fields, names, limit):
```

Comment:
This could use some docs, why would we be overriding field widgets and how do these method parameters affect the outcome?